static/etc/fstab

```diff
@@ -0,0 +1,49 @@
+/run/mnt/base / none bind,x-initrd.mount,ro 0 0
+/run/mnt/data /writable none bind,x-initrd.mount 0 0
+/run/mnt/kernel/modules /usr/lib/modules none bind,x-initrd.mount,ro 0 0
+/run/mnt/kernel/firmware /usr/lib/firmware none bind,x-initrd.mount,ro 0 0
+
+/run/mnt/data/system-data/etc/systemd /etc/systemd none bind,x-initrd.mount 0 0
+/run/mnt/data/system-data/etc/writable /etc/writable none bind,x-initrd.mount 0 0
+/run/mnt/data/system-data/etc/dbus-1 /etc/dbus-1 none bind,x-initrd.mount 0 0
```

Comment:
It would be good to have lines sorted in this file, does not seem to be fully the case

Reply:
Done